src/data/faq_de.json

```diff
@@ -2181,18 +2181,102 @@
         "permalink": "Link zu diesem Eintrag",
         "linkToGlossary": "Link zum Glossar",
         "glossaryWords": {
+            "A": [
+                {
+                    "term": "Android",
+                    "anchor": "android",
+                    "description": "Android ist neben <a href='#glossary_ios'>iOS</a> eines der beiden Betriebssysteme, welche die Corona-Warn-App aktuell unterstützt, und das am weitesten verbreitete mobile Betriebssystem der Welt. Es handelt sich um ein <a href='#glossary_open_source'>Open-Source</a> Betriebssystem für beispielsweise mobile (wie Smartphones oder Tables) aber auch andere Geräte (wie Fernseher oder Autos). Es wird gemeinsam von Google und der Open Handset Alliance (OHA) entwickelt."
+                },
+                {
+                    "term": "Antigen-Schnelltest",
+                    "anchor": "rapid_antigen_test",
+                    "description": "Antigen-Schnelltests ähneln von der Vorgehensweise dem Selbsttest, werden allerdings von geschultem Personal durchgeführt. Eine Gegenüberstellung der beiden Antigentest-Typen wurde vom RKI bereitstellt: <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Downloads/Flyer-Antigentests.pdf?__blob=publicationFile'>Link zur Gegenüberstellung vom RKI</a> <br></br>Nicht alle Antigen-Schnelltest sind ähnlich zuverlässig. Aus diesem Grund hat das Paul-Ehrlich-Institut sog. Mindestkriterien an Antigen-Schnelltests veröffentlicht: <a href='https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests-01-12-2020.pdf?__blob=publicationFile&v=7'>Link zu den Mindestkriterien</a> <br></br>Ein positives Ergebnis eines Antigen-Schnelltests ist noch keine finale Diagnose und sollte durch einen <a href='#glossary_pcr_test'>PCR-Test</a> bestätigt werden."

```

Comment:
Bereit stellt => bereitgestellt

Reply:
Will be changed in the next commit, thanks!